api/api_lib.php

```diff
@@ -40,21 +40,26 @@ function mockSubmissions($courseID, $assignmentID){
        echo $e->getMessage();
     }
 }
+function update_assignment($assignment_params){
+    global $dataMgr;
+    $assignment = $dataMgr->getAssignment(new AssignmentID($assignment_params["assignmentID"]));
 
+    save_assignment($assignment, $assignment_params);
+}
 
+function create_assignment($assignment_params){
+    global $dataMgr;
+    $assignmentType = $assignment_params["assignmentType"];
+    $assignment = $dataMgr->createAssignmentInstance(null, $assignmentType);
+    save_assignment($assignment, $assignment_params);
+}
 
-function createAssignment($assignment_params){
+function save_assignment($assignment, $assignment_params){
```

Comment:
would be nice to have this annotated with a doc. what I think might  make sense is if you have two different functions at the top level - create_assignment, and save_assignment, which call save_assignment with different parameters maybe?